tests/contrib/flask/test_request.py

```diff
@@ -529,10 +529,10 @@ def not_found():
         self.assertEqual(dispatch_span.service, "flask")
         self.assertEqual(dispatch_span.name, "flask.dispatch_request")
         self.assertEqual(dispatch_span.resource, "flask.dispatch_request")
-        self.assertEqual(dispatch_span.error, 1)
-        self.assertTrue(dispatch_span.get_tag("error.msg").startswith("404 Not Found"))
-        self.assertTrue(dispatch_span.get_tag("error.stack").startswith("Traceback"))
-        self.assertEqual(dispatch_span.get_tag("error.type"), "werkzeug.exceptions.NotFound")
+        self.assertEqual(dispatch_span.error, 0)
+        self.assertIsNone(dispatch_span.get_tag("error.msg"))
+        self.assertIsNone(dispatch_span.get_tag("error.stack"))
+        self.assertIsNone(dispatch_span.get_tag("error.type"))
```

Comment:
Note: the handler span propagates the error as well.

Reply:
which should be ok, as long as the main `flask.request` one doesn't have an exception attached, then it'll be ok

Reply:
@brettlangdon it still gets flagged as an error with exception information though which isn't great... 😕

Reply:
the `flask.request` one or the dispatch?

the dispatch one showing an exception should be ok as long as the main `flask.request` one doesn't? or am I missing something

Reply:
So in normal cases of actual endpoints that don't exist, we get the expected behaviour (i.e. `flask.dispatch_request` span does not flag as error). However in the case of `test_404_abort`, if there is an actual endpoint but there's a manual `abort(404)` call, this creates a downstream handler span flagged with an error. Is this acceptable expected behaviour?

Reply:
Got it. I'm ok with it for now. Not great that the handler span will be flagged as erroneous but we can deal with that if requested.

Reply:
oh yeah, that behavior sounds good